scripts/deploy-helm.sh

```diff
@@ -30,9 +30,42 @@ check_if_parameter_is_empty()
   fi
 }
 
-print_help()
+check_api_token()
 {
-   printf "
+  URL=$1
+  if RESPONSE=$(curl -k -s -X POST -d "{\"token\":\"$DYNATRACE_ACCESS_KEY\"}" "$URL/api/v2/apiTokens/lookup" -w "<<HTTP_CODE>>%{http_code}" -H "accept: application/json; charset=utf-8" -H "Content-Type: application/json; charset=utf-8" -H "Authorization: Api-Token $DYNATRACE_ACCESS_KEY"); then
+    CODE=$(sed -rn 's/.*<<HTTP_CODE>>(.*)$/\1/p' <<<"$RESPONSE")
+    RESPONSE=$(sed -r 's/(.*)<<HTTP_CODE>>.*$/\1/' <<<"$RESPONSE")
+    if [ "$CODE" -ge 300 ]; then
+      echo "Failed to check Dynatrace API token permissions - please verify provided URL and API token. $RESPONSE"
+      exit 1
+    fi
+    for scope in "${API_TOKEN_SCOPES[@]}"; do
+      if ! grep -q "$scope" <<< "$RESPONSE"; then
+        echo "Missing permission for the API token: $scope."
+        echo "Please enable all required permissions: ${API_TOKEN_SCOPES[*]} for chosen deployment type: $DEPLOYMENT_TYPE"
+        exit 1
+      fi
+    done
+  else
+      echo "Failed to check Dynatrace API token permissions - please verify provided URL and API token."
```

Comment:
- we could be more specific in message, like "please verify provided ingest url: ${URL}"
- is it the case for connectivity error? then we should not mention TOKEN in this message